src/Mailer/Handler/SendOwnershipMailCommandHandler.php

```diff
@@ -83,24 +85,40 @@ public function sentMail(SendOwnershipRequestedMail $command, string $rawSubject
             return;
         }
 
-        //@todo loop over ALL owners of organisation
-        $ownerId = $ownershipItem->getOwnerId();
-        $ownerDetails = $this->identityResolver->getUserById($ownerId);
-
-        if ($ownerDetails === null) {
-            $this->logger->warning(sprintf('[ownership-mail] Could not load owner details for %s', $ownerId));
+        try {
+            $recipients = $recipientStrategy->getRecipients($ownershipItem);
+        } catch (DocumentDoesNotExist $e) {
+            $this->logger->warning(sprintf('[ownership-mail] Could not load organizer: %s', $e->getMessage()));
```

Comment:
This is a weird message, how can getting the recipients result in a warning when getting the organizer?